core-java-modules/core-java-arrays-convert/src/test/java/com/baeldung/array/conversions/ByteArrayToPrimitiveByteArrayUnitTest.java

```diff
@@ -0,0 +1,38 @@
+package com.baeldung.array.conversions;
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class ByteArrayToPrimitiveByteArrayUnitTest {
```

Comment:
let's explicitly put **public** here and in the test methods

Reply:
Junit5 doesn't require public modifiers on classes and methods and the tests classes nearby also don't have them (some that were written using JU5). But ok, if it works better - I will add the access modifiers.

Reply:
Fixed